### PR TITLE
Pass params for blender

### PR DESCRIPTION
**Pull request by Kataderon**
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/blender_passParams/4/Test_20Report/

### CODE REVIEW

File: vars/rpr_blender28plugin_pipeline.groovy [hunk 1]
```diff
@@ -267,23 +267,23 @@ def executeTestCommand(String osName, Map options)
...
+            run.bat ${options.renderDevice} ${options.testsPackage} \"${options.tests}\" ${options.resX} ${options.resY} ${options.SPU} ${options.iter} ${options.theshold} >> ..\\${options.stageName}.log  2>&1
             """
         }
         break;
     case 'OSX':
         dir("scripts")
         {
             sh """
-            ./run.sh ${options.renderDevice} ${options.testsPackage} \"${options.tests}\" >> ../${options.stageName}.log 2>&1
+            ./run.sh ${options.renderDevice} ${options.testsPackage} \"${options.tests}\" >${options.resX} ${options.resY} ${options.SPU} ${options.iter} ${options.theshold} > ../${options.stageName}.log 2>&1
```
Comment (MagisterDemens) [inline vars/rpr_blender28plugin_pipeline.groovy]: Fix **>>** syntax

File: vars/rpr_blender28plugin_pipeline.groovy [hunk 2]
```diff
@@ -267,23 +267,23 @@ def executeTestCommand(String osName, Map options)
...
             """
         }
         break;
     default:
         dir("scripts")
         {
             sh """
-            ./run.sh ${options.renderDevice} ${options.testsPackage} \"${options.tests}\" >> ../${options.stageName}.log 2>&1
+            ./run.sh ${options.renderDevice} ${options.testsPackage} \"${options.tests}\" >${options.resX} ${options.resY} ${options.SPU} ${options.iter} ${options.theshold} > ../${options.stageName}.log 2>&1
```
Comment (MagisterDemens) [inline vars/rpr_blender28plugin_pipeline.groovy]: Fix **>>** syntax

File: vars/rpr_blender28plugin_pipeline.groovy [hunk 3]
```diff
@@ -883,7 +883,12 @@ def call(String projectBranch = "",
     String tests = "",
     Boolean forceBuild = false,
     Boolean splitTestsExecution = false,
-    Boolean sendToRBS = true)
+    Boolean sendToRBS = true,
+    String resX = '0',
+    String resY = '0',
+    String SPU = '25',
+    String iter = '50',
```
Comment (MagisterDemens) [inline vars/rpr_blender28plugin_pipeline.groovy]: Why SPU < iter ? Max SPU=512, so till iter <= 512 we should set SPU=iter if highest performance is required.
@nlarionov am I right?
P.S. need check this case for other plugins pipelines.

Comment (Kataderon) [reply vars/rpr_blender28plugin_pipeline.groovy]: I got approve that it's right from @nlarionov

Comment (nlarionov) [reply vars/rpr_blender28plugin_pipeline.groovy]: Aproved

**Comment by nlarionov** [comment]
Can be merged

**Pull request merged**